src/components/interfaces/HMI_API.xml

```diff
@@ -4731,12 +4731,42 @@
     </param>
 </function>
 
-
 <function name="OnInteriorVehicleData" functionID="OnInteriorVehicleDataID" messagetype="notification">
   <param name="moduleData" type="Common.ModuleData">
   </param>
 </function>
 
+<function name="ButtonPress" functionID="ButtonPressID" messagetype="request">
```

Comment:
@LitvinenkoIra please compare `ButtonPress` function you are adding with it latest version in [mobile](https://github.com/yang1070/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml) and [hmi](https://github.com/yang1070/sdl_core/blob/master/src/components/interfaces/HMI_API.xml) apis.
For example here in request should be also `appID` parameter and response should be without params at all...